man/networkctl.xml

```diff
@@ -355,6 +355,31 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         which match the file are reconfigured.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>
+          <command>edit</command>
+          <replaceable>FILE</replaceable>|<replaceable>DEVICE</replaceable>…
+        </term>
+        <listitem><para>Edit network configuration files. If no network config file matching
+        the given name is found, a new one will be created under <filename>/etc/</filename>.
```

Comment:
this should say that the filename is supposed to end in ".link", ".network" or ".netdev"